scout/parse/variant/coordinates.py

```diff
@@ -57,22 +57,14 @@ def sv_length(pos, end, chrom, end_chrom, svlen=None):
     return end - pos
 
 
-def sv_end(pos, alt, svend=None, svlen=None):
+def sv_end(pos: int, alt: str, svend: int = None, svlen: int = None) -> int:
     """Return the end coordinate for a structural variant
     The END field from INFO usually works fine, although for some cases like insertions the callers
      set end to same as pos. In those cases we can hope that there is a svlen...
 
     Translocations needs their own treatment as usual
-
-    Args:
-        pos(int)
-        alt(str)
-        svend(int)
-        svlen(int)
-
-    Returns:
-        end(int)
     """
+    svend = svend or pos
```

Comment:
Yes! Only I go nervtous about the order of the alt-`:` version and `svlen`. I think this is fine, but we should test a few different variant types from a few current callers (notably inv and translocation bnd's) to make sure. 😊